src/scss/06_components/paragraphs/_popular-service-item.scss

```diff
@@ -23,33 +25,43 @@ $-service-item-padding: ($wcag-touch-target-size-aaa - get-line-height($-service
 .popular-service-item__links {
   display: flex;
   flex-direction: column;
+  margin-bottom: 0;
   margin-top: $spacing - $-service-item-padding;
+  padding-left: $spacing-and-half;
 
   @include breakpoint($breakpoint-m) {
     margin-top: $spacing-and-half - $-service-item-padding;
   }
 }
 
-.link__style--highlight {
+.popular-service-item__link-wrapper {
+  // The link wrapper needs to have the same font size and line-height as the link itself so that the list marker is
+  // shown in the correct position.
   @include font($-service-item-font);
   margin-bottom: $spacing - 2 * $-service-item-padding;
+  margin-top: 0;
+
+  // The padding here is also to help align the list marker to correct position.
   padding-block: $-service-item-padding;
-  padding-inline-start: calc(var(--line-height) * 1em + $spacing-half);
+  vertical-align: top;
+}
 
-  &::before {
-    @include pseudo-icon('arrow-right', calc(var(--line-height) * 1em), currentColor, inline-block);
-    margin-inline-end: $spacing-half;
-    margin-inline-start: calc(var(--line-height) * -1em - $spacing-half);
-    transform: scaleX(var(--lang-direction, 1));
-  }
+.link__style--highlight {
+  @include font($-service-item-font);
+
+  // Display inline-block helps the external link icon to be displayed directly after the text.
+  display: inline-block;
+
+  // This margin is used to negate the padding below and to position the list marker.
+  margin-block: ($-service-item-padding * -1);
+
+  // The padding below it are used to make the link correct minimum height for the WCAG AAA standard.
+  padding-block: $-service-item-padding;
+  padding-left: $spacing-quarter;
```

Comment:
I changed this now but I think we actually have an issue with this. It seems to be that our compiler converts `padding-inline-start` to `padding-left` and that shouldn't happen because the whole thing stops working if it does that. I checked the compiled css and for me it does that at least. Lets investigate this!